ConsoleView.cpp

```diff
@@ -38,5 +68,6 @@ void ConsoleView::PrintClassRelations(UMLRelationship x){
 //Using both the get_src_class and get_dest_class 
 //@Para: UMLRelationship OBJ
 void ConsoleView::PrintSourceandDest(UMLRelationship x){
```

Comment:
- Again, these method names need to following our naming convention, lowercase_with_underscores.
- print_source_and_dest or print_src_and_dest